File: webtmp/file_list.php
<?php

function eachfile($path){
    $files=[];
    foreach(scandir($path) as $v){
        if($v=="."||$v==".."){
            continue;
        }
        $tpath=$path.$v;
        if(is_dir($tpath)){
            $files=array_merge($files,eachfile($tpath.'/'));
        }else{
            $files[]=$path.$v;
        }
    }
    return $files;
}
$files=eachfile('./');
print_r($files) 

?>

File: webtmp/file_list.py
import os

def eachFile(filepath):
    dirs =  os.listdir(filepath)
    dirs_arr=[]
    for v in dirs:
        child = os.path.join('%s%s' % (filepath, v))
        if os.path.isdir(child):
            dirs_arr.extend(eachFile(child+"/"))
        else:
            dirs_arr.append(child)
    return dirs_arr

if __name__ == '__main__':
    res=eachFile('./')
    # print res
    for v in res:
        # print v.enco
        print v.decode('utf-8')
